tox.ini

```diff
@@ -25,6 +25,13 @@ deps = isort
 skip_install = true
 commands = isort {posargs: -c --diff setup.py strace_process_tree.py tests.py}
 
+[testenv:mypy]
+deps =
+    mypy
+    types-setuptools
+skip_install = true
+commands = mypy --strict .
```

Comment:
I would prefer a `mypy.ini` setting `strict = true`, so I (and my editor integration) can run `mypy` with no additional arguments on .py files and see the same errors that `tox -e mypy` would show.

Reply:
Moved that to `pyproject.toml`.